packages/components/src/components/Image/images.ts

```diff
@@ -28,38 +28,38 @@ export const SVG_IMAGES = {
 export type PngImage = keyof typeof PNG_IMAGES;
 
 export const PNG_IMAGES = {
-    CLOUDY: 'Cloudy.png',
-    CLOUDY_2x: 'Cloudy@2x.png',
-    BACKUP: 'Backup.png',
-    BACKUP_2x: 'Backup@2x.png',
-    CHECK_SHIELD: 'CheckShield.png',
-    CHECK_SHIELD_2x: 'CheckShield@2x.png',
-    CLOCK: 'Clock.png',
-    CLOCK_2x: 'Clock@2x.png',
-    COINS: 'Coins.png',
-    COINS_2x: 'Coins@2x.png',
-    ERROR: 'Error.png',
-    ERROR_2x: 'Error@2x.png',
-    EXTRA_INFO: 'ExtraInfo.png',
-    EXTRA_INFO_2x: 'ExtraInfo@2x.png',
-    FIRMWARE: 'Firmware.png',
-    FIRMWARE_2x: 'Firmware@2x.png',
-    FOLDER: 'Folder.png',
-    FOLDER_2x: 'Folder@2x.png',
-    KEY: 'Key.png',
-    KEY_2x: 'Key@2x.png',
-    PIN: 'Pin.png',
-    PIN_2x: 'Pin@2x.png',
-    PIN_LOCKED: 'PinLocked.png',
-    PIN_LOCKED_2x: 'PinLocked@2x.png',
-    RECOVERY: 'Recovery.png',
-    RECOVERY_2x: 'Recovery@2x.png',
-    UNDERSTAND: 'Understand.png',
-    UNDERSTAND_2x: 'Understand@2x.png',
-    WALLET: 'Wallet.png',
-    WALLET_2x: 'Wallet@2x.png',
-    DONT_DISCONNECT_T: 'DontDisconnectT.png',
-    DONT_DISCONNECT_T_2x: 'DontDisconnectT@2x.png',
-    DONT_DISCONNECT_R: 'DontDisconnectR.png',
-    DONT_DISCONNECT_R_2x: 'DontDisconnectR@2x.png',
+    CLOUDY: 'cloudy.png',
+    CLOUDY_2x: 'cloudy@2x.png',
+    BACKUP: 'backup.png',
+    BACKUP_2x: 'backup@2x.png',
+    CHECK_SHIELD: 'check-shield.png',
+    CHECK_SHIELD_2x: 'check-shield@2x.png',
+    CLOCK: 'clock.png',
+    CLOCK_2x: 'clock@2x.png',
+    COINS: 'coins.png',
+    COINS_2x: 'coins@2x.png',
+    ERROR: 'error.png',
+    ERROR_2x: 'error@2x.png',
+    EXTRA_INFO: 'extra-info.png',
+    EXTRA_INFO_2x: 'extra-info@2x.png',
+    FIRMWARE: 'firmware.png',
+    FIRMWARE_2x: 'firmware@2x.png',
+    FOLDER: 'folder.png',
+    FOLDER_2x: 'folder@2x.png',
+    KEY: 'key.png',
+    KEY_2x: 'key@2x.png',
+    PIN: 'pin.png',
+    PIN_2x: 'pin@2x.png',
+    PIN_LOCKED: 'pin-locked.png',
+    PIN_LOCKED_2x: 'pin-locked@2x.png',
+    RECOVERY: 'recovery.png',
+    RECOVERY_2x: 'recovery@2x.png',
+    UNDERSTAND: 'understand.png',
+    UNDERSTAND_2x: 'understand@2x.png',
+    WALLET: 'wallet.png',
+    WALLET_2x: 'wallet@2x.png',
+    DONT_DISCONNECT_T: 'dont-disconnect-t.png',
```

Comment:
I would love to call it `T2T1`

I cannot call it `..._TT` because `<Image image={`DONT_DISCONNECT_${deviceModel}`} />` it would not work with `T2B1`